lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * View for an individual team.
+ */
+;(function (define) {
+    'use strict';
+    define(['backbone', 'underscore', 'gettext', 'teams/js/views/team_discussion',
+            'text!teams/templates/team-profile.underscore'],
+        function (Backbone, _, gettext, TeamDiscussionView, team_template) {
```

Comment:
Good catch. I didn't mean to use underscores, so I've fixed it to be ``teamTemplate``. Just FYI, I don't name it like a class because the value is a string.